graphdatascience/graph_data_science.py

```diff
@@ -82,16 +85,36 @@ def __init__(
                 None if arrow is True else arrow,
             )
 
-        super().__init__(self._query_runner, namespace="gds", server_version=self._server_version)
+        if auth is not None:
+            if self._query_runner.encrypted():
+                with open(self._path("graphdatascience.resources.field-testing", "pub.pem"), "rb") as f:
+                    pub_key = rsa.PublicKey.load_pkcs1(f.read())
+                self._encrypted_db_password = rsa.encrypt(auth[1].encode(), pub_key).hex()
+
+        self._compute_cluster_ip = None
+
+        super().__init__(self._query_runner, "gds", self._server_version)
+
+    def set_compute_cluster_ip(self, ip: str) -> None:
+        self._compute_cluster_ip = ip
+
+    @staticmethod
+    def _path(package: str, resource: str) -> pathlib.Path:
+        if sys.version_info >= (3, 9):
+            from importlib.resources import files
+
+            # files() returns a Traversable, but usages require a Path object
+            return pathlib.Path(str(files(package) / resource))
+        else:
+            from importlib.resources import path
+
+            # we dont want to use a context manager here, so we need to call __enter__ manually
+            return path(package, resource).__enter__()
 
     @property
     def graph(self) -> GraphProcRunner:
         return GraphProcRunner(self._query_runner, f"{self._namespace}.graph", self._server_version)
 
-    @property
-    def util(self) -> UtilProcRunner:
-        return UtilProcRunner(self._query_runner, f"{self._namespace}.util", self._server_version)
-
```

Comment:
i think thats an error during rebase

Reply:
Thanks for this finding, this PR is definetely not ready for review though.